docs/api-docs/building-checkouts/checkout-customizability.md

```diff
@@ -17,7 +17,7 @@ Once a developer has chosen a path, they can choose the options they want to use
 | | What is it? | When should I use it? | Where do I find it? |
 |-|-|-|-|
 |**Open Checkout**| Open checkout is the open-source option of BigCommerce's Optimized One-Page Checkout.<br/><br/> A browser-based application that converts a cart to an order, allowing shoppers to complete checkout.<br/><br/>  Our reference implementation of checkout built on top of the Checkout JS SDK. | This option allows developers to create a custom checkout from a fully functional foundation (as a starting point for their efforts or even contribute to it).<br/> <br/>Add features or make moderate-sized changes to BigCommerce’s checkout (hosted on the BigCommerce storefront). | [Open Checkout Quick Start](/stencil-docs/customizing-checkout/open-checkout-quick-start)
-|**Checkout JS SDK**| A client-side vanilla JavaScript library that is a wrapper for the Storefront Checkout API.<br/><br/> The SDK handles all the API calls necessary to move shoppers through the checkout experience. From customer login, fetching shipping quotes, and payment methods through to paying an order.| This option allows developers to build a custom checkout from scratch by creating a presentation layer for checkout. Completely replacing the UI/UX without having to do the heavy lifting of dealing with Storefront API endpoints.<br/><br/> Significant front-end customizations and/or feature additions to checkout (hosted on the BigCommerce storefront).<br/><br/> **Note:** SDK does not let you introduce new behavior. While you can filter based on shipping methods, you cannot create new shipping methods or payment methods. | [Checkout SDK](/stencil-docs/customizing-checkout/checkout-sdk) |
+|**Checkout JS SDK**| A client-side vanilla JavaScript library that is a wrapper for the Storefront Checkout API.<br/><br/> The SDK handles all the API calls necessary to move shoppers through the checkout experience. From customer login, fetching shipping quotes, and payment methods through to paying an order.| This option allows developers to build a custom checkout from scratch by creating a presentation layer for checkout. Completely replacing the UI/UX without having to do the heavy lifting of dealing with Storefront API endpoints.<br/><br/> Significant front-end customizations and/or feature additions to checkout (hosted on the BigCommerce storefront).<br/><br/> **Limitations:** <ul><li> SDK does not let you introduce new behavior. While you can filter based on shipping methods, you cannot create new shipping methods or payment methods. </li> <li> Hosted forms do not work with stencil localhost since forms are served over https and stencil only supports http. Developers must test the order confirmation flow on stencil using offline payment methods.</li></ul> | [Checkout SDK](/stencil-docs/customizing-checkout/checkout-sdk) |
```

Comment:
```suggestion
|**Checkout JS SDK**| A client-side vanilla JavaScript library that is a wrapper for the Storefront Checkout API.<br/><br/> The SDK handles all the API calls necessary to move shoppers through the checkout experience. From customer login, fetching shipping quotes, and payment methods through to paying an order.| This option allows developers to build a custom checkout from scratch by creating a presentation layer for checkout. Completely replacing the UI/UX without having to do the heavy lifting of dealing with Storefront API endpoints.<br/><br/> Significant front-end customizations and/or feature additions to checkout (hosted on the BigCommerce storefront).<br/><br/> **Limitations:** <ul><li> SDK does not let you introduce new behavior. While you can filter based on shipping methods, you cannot create new shipping methods or payment methods. </li> <li> Hosted forms do not work with Stencil localhost since forms are served over https and Stencil only supports http. Developers must test the order confirmation flow on Stencil using offline payment methods.</li></ul> | [Checkout SDK](/stencil-docs/customizing-checkout/checkout-sdk) |
```